posts/a-short-history-of-code-search.md

```diff
@@ -4,22 +4,22 @@ authorSlug: lisa-tagliaferri
 authorDisplayName: Lisa Tagliaferri
 tags: [tutorial, code search]
 publicationDate: January 3, 2022
-description: Learn how to make a Composer package
```

Comment:
`description` and `browserTitle` were copy-pasted and not updated, so I am fixing them here!